x-pack/plugins/ml/public/application/components/field_stats_flyout/eui_combo_box_with_field_stats.tsx

```diff
@@ -6,12 +6,205 @@
  */
 
 import type { FC } from 'react';
-import React, { useMemo } from 'react';
-import type { EuiComboBoxProps } from '@elastic/eui/src/components/combo_box/combo_box';
-import type { EuiComboBoxOptionOption } from '@elastic/eui';
-import { EuiComboBox } from '@elastic/eui';
+import React, { useEffect, useMemo, useState } from 'react';
+import type { EuiSelectableOption } from '@elastic/eui';
+import {
+  EuiPopoverFooter,
+  useEuiBackgroundColor,
+  EuiFilterButton,
+  EuiFilterGroup,
+  EuiInputPopover,
+  htmlIdGenerator,
+  EuiSelectable,
+  EuiSwitch,
+  EuiProgress,
+  EuiIcon,
+  EuiBadge,
+  EuiFlexItem,
+} from '@elastic/eui';
 import { css } from '@emotion/react';
+import { i18n } from '@kbn/i18n';
+import { isDefined } from '@kbn/ml-is-defined';
+import type { Field } from '@kbn/ml-anomaly-utils';
+import type { Aggregation } from '../../../../server/shared';
 import { useFieldStatsTrigger } from './use_field_stats_trigger';
+import { useCurrentThemeVars } from '../../contexts/kibana';
+import { useFieldStatsFlyoutContext } from './use_field_stats_flytout_context';
+
+const MIN_POPOVER_WIDTH = 300;
```

Comment:
The current AD wizard control has a width of 400px from the looks of it, so not sure if you want to make this 400, or alternatively increase the width for the selects in the AD wizard?

Reply:
Updated in latest changes